scripts/Install.ps1

```diff
@@ -0,0 +1,115 @@
+# Automated Install script by Midonei
+$Host.UI.RawUI.WindowTitle = "Installing MagiskOnWSA..."
+function Test-Administrator {
+    [OutputType([bool])]
+    param()
+    process {
+        [Security.Principal.WindowsPrincipal]$user = [Security.Principal.WindowsIdentity]::GetCurrent();
+        return $user.IsInRole([Security.Principal.WindowsBuiltinRole]::Administrator);
+    }
+}
+
+function Get-InstalledDependencyVersion {
+    param (
+        [string]$Name,
+        [string]$ProcessorArchitecture
+    )
+    process {
+        return Get-AppxPackage -Name $Name | ForEach-Object { if ($_.Architecture -eq $ProcessorArchitecture) { $_ } } | Sort-Object -Property Version | Select-Object -ExpandProperty Version -Last 1;
+    }
+}
+
+function Finish {
+    Clear-Host
+    Start-Process "wsa://com.topjohnwu.magisk"
+    Start-Process "wsa://com.android.vending"
+}
+
+If (-Not (Test-Administrator)) {
+    Set-ExecutionPolicy -Scope CurrentUser -ExecutionPolicy Bypass -Force
+    $proc = Start-Process -PassThru -WindowStyle Hidden -Verb RunAs ConHost.exe -Args "powershell -ExecutionPolicy Bypass -Command Set-Location '$PSScriptRoot'; &'$PSCommandPath' EVAL"
+    $proc.WaitForExit()
+    If ($proc.ExitCode -Ne 0) {
+        Clear-Host
+        Write-Warning "Failed to launch start as Administrator`r`nPress any key to exit"
+        $null = $Host.UI.RawUI.ReadKey('NoEcho,IncludeKeyDown');
+    }
+    exit
+}
+ElseIf (($args.Count -Eq 1) -And ($args[0] -Eq "EVAL")) {
+    Start-Process ConHost.exe -Args "powershell -ExecutionPolicy Bypass -Command Set-Location '$PSScriptRoot'; &'$PSCommandPath'"
+    exit
+}
+
+If (((Test-Path -Path "Styles","vendor.img","WsaProxy","system.img","CustomInstall","AppxManifest.xml","lxutil.dll","Microsoft.Web.WebView2.Core.dll","wslcoredeps.dll","Licenses","WsaClient","WSACodecs.dll","classicAppInstaller_WSA.sccd","userdata.vhdx","system_ext.img","networking.json","networking_schema.json","Images","libEGL.dll","WsaSettings.winmd","appcompatdb_schema.json","product.img","WsaService","Fonts","appcompatdb.json","Microsoft.Web.WebView2.Core.winmd","Microsoft.UI.Xaml_x64.appx","metadata.vhdx","WsaSettingsBroker","libGLESv2.dll","WSACrashUploader","wsldevicehost.dll","Tools","Microsoft.UI.Xaml.winmd","Registry.dat","GSKServer","WsaSettings.exe","Assets","wslcore.dll","Microsoft.VCLibs.x64.14.00.Desktop.appx","gfxstream_backend.dll","wslhost.exe","resources.pri") -Eq $false).Count) {
```

Comment:
We shouldn't write a dead file list, it would be troublesome if the updated version files change, and this doesn't take into account the arm64 version.

Reply:
This is exactly the script we're writing now. I guess that could go in another PR.

Reply:
> This is exactly the script we're writing now. 

This list of files is generated dynamically

Reply:
Sounds good to you if we remove the check altogether?

Reply:
No